src/components/Header.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import styled from 'styled-components/macro'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import {
+  faGithub,
+  faLinkedin,
+  faStackOverflow
+} from '@fortawesome/free-brands-svg-icons'
+import { InnerWrapper, OuterWrapper } from './global/GlobalStyling';
+import Annika from '../assets/Annika.jpg'
+import BackgroundImg from '../assets/BackgroundImg.jpg'
+
+export const Header = () => {
+  const HeaderBackground = styled.header`
+    position: relative;
+    height: 70vh;
+    width: 100%;
+    background: linear-gradient(rgba(82, 81, 81, 0.5), rgba(0, 0, 0, 0.5)), url(${BackgroundImg});
```

Comment:
Clever way of changing the look of the background image right in the browser! Looks really nice